packages/core-sdk/src/client.ts

```diff
@@ -73,17 +70,13 @@ export class StoryClient {
   }
   /**
```

Comment:
A small one, but I think we should be consistent regarding spacing, in this case there's no space between the end of the previous function and the comments.

Reply:
we should use eslint to enforce this. no need to address it in this pr